### PR TITLE
Style

### PR DESCRIPTION
**Pull request by Dachrono**
- Add gamming style
- Change title size
- Change button position

### CODE REVIEW

File: src/style/index.css [hunk 1]
```diff
@@ -1,9 +1,24 @@
...
+}
+
+h1 {
+  background-color: #dd8512e3;
+  padding: 0 0 0 20px;
+}
+
+h3 {
+  background-color: #f0f8ffcc;
+  padding: 10px 30px 10px 30px;
```
Comment (usorfaitheloho) [inline src/style/index.css]: You did well with your Leaderboard design 👍 , I'd kindly suggest that you make the heading `Leaderboard` to be quite bigger than it already is, and also align the submit button at the end as it is in the framework given as it was stated that you should follow the design framework but add your designs 😄 

Your design :
image

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @Dachrono  ,
> 
> Great job for implementing some of the Project Requirement, but there are some changes that need to be implemented in order to proceed to the next project 👍
> 
> ## Highlights 👏🏻
> ✔️ Good UI ✔️ You used correct Git Hub-Flow.✔️ Good Commit Messages
> 
> almost 

> ## Required Changes ♻️
> _Check the comments below the review._
> 
> * [x]  Kindly update your README file to add tools to be installed to run your project and how your project can be started on a browser 
> * [x]  Please , also edit your PR to highlight activities carried out in this milestone. Always remeber to do this for all projects yor work on 😄 
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/style/index.css (hunk 1 above)
Comment (Dachrono) [reply src/style/index.css]: Solved

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @Dachrono 

Your project is complete! Great job for nailing it 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request ✔️ 
- Wireframe design followed✔️ 
- App works as expected✔️ 
- No linter errors✔️ 

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

tufoin

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**